.gitignore

```
@@ -47,7 +47,7 @@ formplayer.properties
 application.properties
 formplayer.jar
 .vscode/*
-
+.env
```

This will match any file in the repo name `.env`. Should it be constrained to the top-level?
```suggestion
/.env
```